ibm/mas_devops/roles/suite_install/templates/airgap-catalogs.yml.j2

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
- Anything airgap related should go into the mas_airgap extension that is being worked on.
- Catalog installation is performed in ocp_setup_mas_deps role already

Reply:
Have removed the airgap extension from suite_install